torch/utils/_mem_tracker_utils.py

```diff
@@ -0,0 +1,46 @@
+from typing import NamedTuple, Union
+
```

Comment:
Why is this a separate file?

Reply:
Using all of these things in FSDP module tracker as well. Also, these are very general utils which were cluttering the main file.